package.json

```diff
@@ -26,9 +26,9 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.6.0",
-    "@aurodesignsystem/auro-calendar": "^1.0.0",
-    "@aurodesignsystem/auro-dropdown": "^2.1.1",
-    "@aurodesignsystem/auro-input": "^2.8.0",
+    "@aurodesignsystem/auro-calendar": "^1.0.3",
```

Comment:
Missed one. 

`"@alaskaairux/design-tokens": "^3.7.0",`

Reply:
@blackfalcon I assume you just mean updating to the latest version?

Reply:
Current is `v3.9.1`. 

If it can be updated, that's preferred, but I was referencing that the dev dependency is still `"@alaskaairux/design-tokens": "^3.7.0",` and the peer dependency is `"@alaskaairux/design-tokens": "^3.6.0",`